walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -19,24 +19,87 @@
 import java.util.Collection;
 import java.util.Collections;
 import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
 
-/**
- * Given a list of events, returns a list of possible time ranges for the meeting request.
- */
 public final class FindMeetingQuery {
+
+  /** Create a meeting timeslot if there is enough time between firstTime and secondTime. */
+  private Optional<TimeRange> createDuration(long duration, int firstTime, int secondTime, 
```

Comment:
this can be static since its a pure function I believe?

https://docs.google.com/document/d/12ymBTyMj3NJWlAKQUky_5DL4AZF9rG97Q9sPhrYrYN4/edit?hl=en#heading=h.1lhx7i5t3aqo